webextension/scripts/utils.js

```diff
@@ -133,7 +133,8 @@ const excluded_urls = [
   'moz-extension:',
   '192.168.',
   '10.',
-  'file:'
+  'file:',
+  'edge:'
```

Comment:
You also can add a new case in `isValidUrl` and `isNotExcludedUrl` in utils.spec.js

Reply:
Okay, will do it. Thanks :-)

Reply:
I added the tests. Thank you!